SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Sandstorm operates on an evergreen release model. Only the latest release is considered supported.
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities to security@sandstorm.io for responsible disclosure. We ask that you hold disclosure for at least 90 days from disclosure or 48 hours from patch release, whichever is shorter.
```

Comment:
Ideally the fix will be in the next patch release, but what if it is not?  Would it be more clear to say "48 hours after patch release which incorporates the fix"?

Reply:
Yeah, I didn't say "next release", but "patch release", which I meant to mean: "The release of the patch for the issue". I agree that I can and should be clearer on that.

Reply:
"hold disclosure for at least 90 days from disclosure" also felt particularly unclear on a re-read. I've adjusted the language to, I think, be more clear all around.